<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_2</class>
 <widget class="QDialog" name="fct_comp_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des places d'une catégorie (version 2)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBox_categorie">
     <item>
      <property name="text">
       <string>Choisir une catégorie</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_fct_comp_2">
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>noPlace</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noRang</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>noZone</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_fct_comp_2">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>refreshResult()</slot>
 </slots>
</ui>
